map/bookmark_manager.cpp

```diff
@@ -2976,6 +2977,132 @@ UserMarkLayer * BookmarkManager::GetGroup(kml::MarkGroupId groupId)
   return it != m_categories.end() ? it->second.get() : nullptr;
 }
 
+struct QuadtreeElement
+{
+  int m_rank;
+  Bookmark * m_bm;
+  uint64_t m_zCurveCoord = 0; // z-order curve coordiante
+
+  bool operator < (QuadtreeElement const & rhs) const
+  {
+    return m_zCurveCoord < rhs.m_zCurveCoord;
+  }
+
+  static bool rankLess(QuadtreeElement const & lhs, QuadtreeElement  const & rhs)
+  {
+    return lhs.m_rank < rhs.m_rank;
+  }
+};
+
+using Quadtree = std::vector<QuadtreeElement>;
+
+static void SetMinZoom(Quadtree & quadtree)
```

Comment:
why do you need static here?

Reply:
to suppress warnings like `function call without prototype`

Reply:
Also I don't want the function to have external linkage